recipes/bluemath-tk/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "bluemath-tk" %}
+{% set version = "1.1.8" %}
+{% set sha256 = "201b346a325a1b2c0ecabc07f06b06df15c181fb0b279a6ab45ab5e1ef549d2a" %}
+{% python_min = 3.11 %}
```

```suggestion
{% set python_min = 3.11 %}
```
Sorry I forgot to add `set` to the previous suggestion.  That should fix your error.